ff-netF48-server-sdk.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net48</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <RootNamespace>io.harness.cfsdk</RootNamespace>
```

Comment:
Tested by changing sample app target and confirming that it works for both .net core and framework